pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -312,6 +311,26 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
+func getDruidDeployCommands(config *config.GardenletConfiguration) []string {
+	command := []string{"" + "/bin/etcd-druid"}
+	command = append(command, "--enable-leader-election=true")
+	command = append(command, "--ignore-operation-annotation=false")
+	command = append(command, "--workers=50")
+	command = append(command, "--compaction-workers=3")
```

Comment:
Since we now have a nice configuration struct, do you want to make these two fields configurable as well? :)

Reply:
I took care of this